src/server.rs

```diff
@@ -335,7 +334,14 @@ pub fn serve(args: &Args) -> BoxResult<()> {
     let mut events = Events::with_capacity(32);
 
     while is_alive() {
-        poll.poll(&mut events, Some(POLL_TIMEOUT))?;
+        if let Err(err) = poll.poll(&mut events, Some(POLL_TIMEOUT)) {
+            if err.kind() == std::io::ErrorKind::Interrupted {
+                log::debug!("Poll interrupted: \"{err}\", ignored, continue polling");
```

Comment:
This message isn't consistent with other logging statements in this project: they all start with a lower-case letter and employ "<state>, <infinitive-action>" grammar, rather than "<state>, <declarative-decision>"

Perhaps `"poll interrupted, \"{err}\" ignored; resuming poll"`